src/jit/gentree.cpp

```diff
@@ -8446,11 +8446,17 @@ GenTree** GenTreeUseEdgeIterator::GetNextUseEdge() const
             }
 
         // Call, phi, and SIMD nodes are handled by MoveNext{Call,Phi,SIMD}UseEdge, repsectively.
+        //
+        // If FEATURE_MULTIREG_ARGS is enabled, so PUTARG_STK nodes also have special handling.
         case GT_CALL:
         case GT_PHI:
 #ifdef FEATURE_SIMD
         case GT_SIMD:
 #endif
+#if FEATURE_MULTIREG_ARGS
+        case GT_PUTARG_STK:
+#endif
+
```

Comment:
Well maybe you can add a comment say that these cases are consider error cases because they should have been handled specially in GenTreeUseEdgeIterator::operator++()